src/main/java/com/internet/shop/service/OrderService.java

```diff
@@ -0,0 +1,19 @@
+package com.internet.shop.service;
+
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import java.util.List;
+
+public interface OrderService {
+    Order create(Order order);
```

Comment:
we don't need create() method on service layer
https://mate-academy.github.io/jv-program-fulltime/02_jdbc_and_servlets/homework/lesson_14_hw2.html